pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -526,6 +527,13 @@ def download(date_array, data_path, tag='', inst_id='', supported_tags=None,
                                      start=date_array[0],
                                      stop=date_array[-1])
 
+    # Create temproary directory if files need to be unzipped.
```

Comment:
Can you add a comment above line 518 that summarizes briefly what's in `inst_dict`?  Doesn't need to list all possible keys, just say in a sentence what it is.